runtime/common/src/slots.rs

```diff
@@ -659,16 +673,14 @@ impl<T: Config> Module<T> {
 					<Onboarding<T>>::insert(&para_id, entry);
 				}
 				None => {
-					// For renewals, reserve any extra on top of what we already have held
-					// on deposit for their chain.
+					// For renewals, reserve as much as we need for this deposit or any future deposit.
+					let deposit_held = Self::deposit_held(&para_id);
 					let extra = if let Some(additional) =
-						amount.checked_sub(&Self::deposit_held(&para_id))
+						amount.checked_sub(&deposit_held)
 					{
-						if T::Currency::withdraw(
+						if T::Currency::reserve(
 							&para_id.into_account(),
-							additional,
-							WithdrawReasons::FEE,
-							ExistenceRequirement::AllowDeath
+							amount.max(deposit_held),
 						).is_err() {
```

Comment:
The logic here actually changes, and while it makes more sense now, I am not sure how it was working before. I think there is or was a bug...

Reply:
i don't think this is correct. it's meant to withdraw/reserve just the extra, if any, to brgin the total held for the slot up to the new bid. this new code would reserve not just the extra, but also the original amount too (which is already reserved from before).

Reply:
ill have a second look. At the time, I justified things were right in my head, but might have been mistaken